.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'm usually using only ESLint for my vue projects, but if prettier is more common I don't mind